mvnxparser/mvnxStreamReader/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+# Copyright: (C) 2017 iCub Facility
+# Author: Diego Ferigo
+# CopyPolicy: Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
+
+# Load dependencies
+find_package(Qt5Xml REQUIRED)
+
+# Build the libraries
+add_library(mvnxStreamReader STATIC mvnxStreamReader.cpp)
+
+# Link the libraries used by this library
+target_link_libraries(mvnxStreamReader xmlStreamReader)
+qt5_use_modules(mvnxStreamReader Xml)
+
+# Export the includes needed to who inherits this library
+target_include_directories(mvnxStreamReader INTERFACE ${CMAKE_CURRENT_SOURCE_DIR})
```

Comment:
Consider that the INTERFACE include directories will be propagated to all the target linking to this target, even when it is installed. I suggest to guard the use of this local include directory with a BUILD specific generator expression.